src/VisualStudio/Core/Def/ID.RoslynCommands.cs

```diff
@@ -25,6 +25,7 @@ public static class RoslynCommands
             public const int SetSeverityHidden = 0x0113;
             public const int SetSeverityNone = 0x0114;
             public const int OpenDiagnosticHelpLink = 0x0116;
+            public const int SetActiveRuleSet = 0x0118;
```

Comment:
0x0115 and 0x0117 are IDs for menu groups. They're used in the .vsct file to put the menu items in the right places, but the language services don't make use of them.